index.bs

```diff
@@ -2807,7 +2753,7 @@ On a given [=interface=], there must exist at most one stringifier.
         };
     </pre>
 
-    Thus, prose is required to explain the stringification behavior.
+    Thus, prose needs to explain the stringification behavior.
```

Comment:
```suggestion
    Thus, there needs to be prose to explain the stringification behavior.
```

I read this as an existential: "_some_ prose is required to explain the stringification behavior". The changed version doesn't allow that reading, and so doesn't sound very idiomatic.